src/components/Operatives/OperativeWorkOrderListItem.js

```diff
@@ -1,24 +1,30 @@
 import PropTypes from 'prop-types'
+import Link from 'next/link'
 
-const OperativeWorkOrderListItem = ({ operativeWorkOrder }) => (
-  <>
-    <li className="green-info-box">
-      <div className="appointment-details">
-        <p className="lbh-body lbh-!-font-weight-bold text-white">{`${operativeWorkOrder.appointment.start}-${operativeWorkOrder.appointment.end}`}</p>
+const OperativeWorkOrderListItem = ({ operativeWorkOrder, index }) => {
+  return (
+    <>
+      <li data-id={index} className="green-info-box">
+        <div className="appointment-details">
+          <p className="lbh-body lbh-!-font-weight-bold text-white">{`${operativeWorkOrder.appointment.start}-${operativeWorkOrder.appointment.end}`}</p>
 
-        <p className="lbh-body lbh-!-font-weight-bold text-white">
-          {operativeWorkOrder.priority}
-        </p>
-      </div>
-      <p className="middle-right">
-        <i className="arrow right"></i>
-      </p>
-    </li>
-  </>
-)
+          <p className="lbh-body lbh-!-font-weight-bold text-white">
+            {operativeWorkOrder.priority}
+          </p>
+        </div>
+        <div className="middle-right">
+          <Link href={`/work-orders/${operativeWorkOrder.reference}`}>
+            <a className="arrow right" id="arrow-right"></a>
```

Comment:
We should avoid adding an `id` as this element is in a list and ids should appear in the document only once.